pkg/types/azure/platform.go

```diff
@@ -36,8 +36,31 @@ type Platform struct {
 	//
 	// +optional
 	ComputeSubnet string `json:"computeSubnet,omitempty"`
+
+	// CloudName is the name of the Azure cloud environment which can be used to configure the Azure SDK
+	// with the appropriate Azure API endpoints.
+	// If empty, the value is equal to `AzurePublicCloud`.
+	// +optional
+	CloudName CloudEnvironment `json:"cloudName,omitempty"`
```

Comment:
`explain` was recently added and I think you might need to generate some code for this: https://github.com/openshift/installer/blob/master/docs/dev/explain.md

Reply:
Ah, nice!